CordovaDemo/platforms/ios/www/plugins/pspdfkit-cordova-ios/PSPDFKitPlugin/pspdfkit.js

```diff
@@ -33,9 +33,9 @@ var PSPDFKitPlugin = new function() {
                         }
                     }
                     cordova.exec(function (result) {
-                        if (callback) callback(result);
+                        if (callback) callback(result, null);
                     }, function (error) {
-                        alert(error);
+                        if (callback) callback(null, error); else console.log(error);
```

Comment:
It feels a bit weird to only log the error if no callback was provided.
What do you think about always logging the error, no matter if a callback was provided or not?

Reply:
I feel both is valid - logging rules for Cordova are not as well defined. 
Probably easier for us if we change it to always log, support-wise.

Reply:
This change was triggered by Z#14617. Before we were showing an alert with the error and it caused very strange issues in Ionic, but not Cordova.

Reply:
Yeah, I agree that we shouldn't show the error in an alert, but a console log can't really hurt, can it?

Reply:
You mean to always anyway and do the callback?

Reply:
Yes, exactly. (Well, the callback only if it was provided, just as we do now.)

Reply:
Thank you! Done in https://github.com/PSPDFKit/Cordova-iOS/pull/84/commits/9ce061e24f8631ad13c962442c05388893ce2302